src/emulator/rtc/include/rtc/rtc.h

```diff
@@ -17,12 +17,16 @@
 
 #pragma once
 
-#include <host/state.h>
 #include <psp2/types.h>
 
 #include <chrono>
 #include <cstdint>
 
+#ifdef WIN32
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#endif
```

Comment:
Would it be possible to forward declare FILETIME rather than include windows.h?

That horse has already bolted, but windows.h increases compile times quite a bit.

Reply:
I know, unfortunately I tried it and it didn't work. I don't remember the error.

Reply:
Maybe yeah.

https://msdn.microsoft.com/en-us/library/windows/desktop/ms724284(v=vs.85).aspx
This suggests the actual struct is called `_FILETIME` and the new type is called `FILETIME`. So it might work if you add the leading underscore. If you can be bothered trying it, go for it. Otherwise it's fine.

Reply:
that fixed it 👍